.github/workflows/build.yml

```diff
@@ -121,108 +117,84 @@ jobs:
     - name: checkout ouroboros-network repository
       uses: actions/checkout@v2
 
+    # Build dependencies
+    - name: Build dependencies
+      run: cabal --builddir="$CABAL_BUILDDIR" configure --enable-tests
+
     - name: Use cabal.project.local.windows
-      run: cp ./cabal.project.local.ci.windows ./cabal.project.local
+      run: |
+        cat ./cabal.project.local.ci.windows >> ./cabal.project.local
```

Comment:
Is there something wrong with `cp`?

Reply:
I do a `configure` in he previous step, so here I'm appending to whatever `local` file that `configure` produced.

Reply:
I see, thanks for pointing this out.